.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+/tmp
```

Comment:
Why put it inside .gitignore? Maybe we could generate this directly inside `generated-contracts` folder? Although this would probably require a structure change (to create subdirectories) so no need to do it now.

Reply:
Oh this is in the project root dir. The `/tmp` root directory is something I got used to with Ruby on Rails. I sometimes use it to keep some temporary project related files